### PR TITLE
Adding aoai-data-validation environment

### PR DESCRIPTION
**Pull request by rjaincc**
Adding aoai-data-validation environment

### CODE REVIEW

File: assets/training/aoai/data_validation/environment/context/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,20 @@
+FROM mcr.microsoft.com/azureml/openmpi4.1.0-ubuntu20.04:{{latest-image-tag}}
+
+
+RUN apt-get update -y && apt-get install binutils libssh-4 libsqlite3-0 libpam-modules linux-libc-dev libldap-common libldap-2.4-2 libc-bin libc-dev-bin libc6 libc6-dev libcurl3-gnutls libgnutls30 tar openssh-server openssh-client openssl curl -y
+
+WORKDIR /
+
+ENV CONDA_PREFIX=/azureml-envs/python-sdk-v2
```
Comment (vizhur) [inline assets/training/aoai/data_validation/environment/context/Dockerfile:8]: it's not python-sdk-v2, right?

File: assets/training/aoai/data_validation/environment/context/conda_dependencies.yaml [hunk 2]
```diff
@@ -0,0 +1,19 @@
+name: python-sdk-v2
```
Comment (vizhur) [inline assets/training/aoai/data_validation/environment/context/conda_dependencies.yaml:1]: probably should be named differently

**Comment by vizhur** [comment]
`safe to publish` label is needed for the new asset onboard only **should** be set by assets PM or Dev only

**Comment by rjaincc** [comment]
Internal release. Closing the PR.

**Pull request closed without merging**